cmdLine/src/main/scala/topgun/cmdline/JfrReader.scala

```diff
@@ -64,15 +61,23 @@ class JfrReader(cmdLine: JfrParseCommandLine) {
     FileWriter.closeAllFiles
     System.out.println(s"Found                         ${totals.totalEvents} events")
     System.out.println(s"Ignored                       ${totals.ignoredEvents} events")
+
+    System.out.println(s"Found Native                         ${totals.totalEventsNative} events")
```

Comment:
align the text - somethinglike
```suggestion
    System.out.println(s"Found Native                 ${totals.totalEventsNative} events")
```

Reply:
Done.